sdf/1.7/inertial.sdf

```diff
@@ -6,9 +6,9 @@
     <description>The mass of the link.</description>
   </element>
 
-  <include filename="pose.sdf" required="0">
-    <description>This is the pose of the inertial reference frame, relative to the specified reference frame. The origin of the inertial reference frame needs to be at the center of gravity. The axes of the inertial reference frame do not need to be aligned with the principal axes of the inertia.</description>
-  </include>
+  <element name="pose" type="pose" default="0 0 0 0 0 0" required="0">
```

Comment:
Can you change the 1.8 spec instead?

Reply:
Fixed c3cfff6e8954aa5b235f65cf136d73eeb677255f